assets/js/wpjm-stats.js

```diff
@@ -4,7 +4,148 @@ import domReady from '@wordpress/dom-ready';
 import { createHooks } from '@wordpress/hooks';
 
 ( function () {
```

Comment:
Just a heads up this top-level wrapper is not needed, with webpack/modules-based code top-level variables/functions in the file won't be globals, but local to the file.

Reply:
Oh and on that note it's also easy to break out code to separate files and import them as the code is starting to get bigger, they will be packaged into one file (the entry point specified in the webpack config, `wpjm-stats.js` in this case.)

Reply:
done